neo/io/nwbio.py

```diff
@@ -611,23 +658,54 @@ def _write_spiketrain(self, nwbfile, spiketrain):
 
     def _write_event(self, nwbfile, event):
         hierarchy = {'block': event.segment.block.name, 'segment': event.segment.name}
-        tS_evt = AnnotationSeries(
-            name=event.name,
-            data=event.labels,
-            timestamps=event.times.rescale('second').magnitude,
-            description=event.description or "",
-            comments=json.dumps(hierarchy))
+        # if constant timestamps
+        timestamps = event.times.rescale('second').magnitude
+        if any(timestamps) == any(timestamps):
+            tS_evt = TimeSeries(name=event.name,
+                                data=event.labels,
+                                starting_time=0.0,
+                                rate=0.01,
+                                unit=str(event.units),
+                                description=event.description or "",
+                                comments=json.dumps(hierarchy))
+        else:
+            tS_evt = TimeSeries(name=event.name,
+                                data=event.labels,
+                                timestamps=event.times.rescale('second').magnitude,
+                                unit=str(event.units),
+                                description=event.description or "",
+                                comments=json.dumps(hierarchy))
+
         nwbfile.add_acquisition(tS_evt)
         return tS_evt
 
-    def _write_epoch(self, nwbfile, epoch):
+    def _write_manage_epoch(self, nwbfile, segment, epoch, arr):
```

Comment:
This method could benefit from having a docstring. What exactly is `arr` here?

Reply:
arr is an epoch array with t_start and t_stop sorted in ascending order.
I don't think it's necessary to create a docstring for this function. It's just an intermediate step in order to respect NWBInspector.

Reply:
If you sort t_start and t_stop independently, doesn't this corrupt data in case one epoch duration is contained in another?